<!DOCTYPE html>
<html lang="en"> 
<head>
    <title>OncoGrapher Documentation</title>
    
    <!-- Meta -->
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    
    <meta name="description" content="OncoGrapher Docs">
    <!-- <meta name="author" content="Xiaoying Riley at 3rd Wave Media">     -->
    <link rel="shortcut icon" href="favicon.ico"> 
    
    <!-- Google Font -->
    <link href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700&display=swap" rel="stylesheet">
    
    <!-- FontAwesome JS-->
    <script defer src="assets/fontawesome/js/all.min.js"></script>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">

    <!-- Plugins CSS -->
    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.2/styles/atom-one-dark.min.css">

    <!-- Theme CSS -->  
    <link id="theme-style" rel="stylesheet" href="assets/css/theme.css">

</head> 

<body class="docs-page">    
    <header class="header fixed-top">	    
        <div class="branding docs-branding">
            <div class="container-fluid position-relative py-2">
                <div class="docs-logo-wrapper">
					<button id="docs-sidebar-toggler" class="docs-sidebar-toggler docs-sidebar-visible mr-2 d-xl-none" type="button">
	                    <span></span>
	                    <span></span>
	                    <span></span>
	                </button>
									<div class="site-logo"><a class="navbar-brand" href="index.html"><img class="logo-icon mr-2" src="assets/images/logo.png" style="height:57.46px;width:80px;" alt="logo"><span class="logo-text">OncoGrapher<span class="text-alt">Docs</span></span></a></div>    
									
                </div><!--//docs-logo-wrapper-->
	            <div class="docs-top-utilities d-flex justify-content-end align-items-center">
	                <!-- <div class="top-search-box d-none d-lg-flex">
		                <form class="search-form">
				            <input type="text" placeholder="Search the docs..." name="search" class="form-control search-input">
				            <button type="submit" class="btn search-btn" value="Search"><i class="fas fa-search"></i></button>
				        </form>
	                </div> -->
	
					<ul class="social-list list-inline mx-md-3 mx-lg-5 mb-0 d-none d-lg-flex">
						<!-- <li class="list-inline-item"><a href="https://github.com/pughlab/crescent-frontend" target="_blank" rel="noopener noreferrer"><i class="fab fa-github fa-fw"></i></a></li>
						<li class="list-inline-item"><a href="https://doi.org/10.1093/nar/gkaa437" target="_blank" rel="noopener noreferrer"><i class="fas fa-book mr-2"></i></a></li>
		                <li class="list-inline-item"><a href="https://youtube.com/playlist?list=PLdXeBakJd4aLGpi2OPWTNeet2jb0iXfPJ" target="_blank" rel="noopener noreferrer"><i class="fab fa-youtube"></i></a></li> -->
		            </ul><!--//social-list-->
					<a href="https://app.oncographer.ca/" target="_blank" rel="noopener noreferrer" class="btn btn-primary"><i class="fas fa-play-circle mr-2"></i>App (UHN VPN only)</a>
					<!-- <a href="#" class="btn btn-info"><i class="fas fa-play-circle mr-2"></i> Go to crescent.cloud</a> -->

	            </div><!--//docs-top-utilities-->
            </div><!--//container-->
        </div><!--//branding-->
    </header><!--//header-->
    
    <div class="docs-wrapper">
	    <div id="docs-sidebar" class="docs-sidebar">
		    <div class="top-search-box d-lg-none p-3">
                <form class="search-form">
		            <input type="text" placeholder="Search the docs..." name="search" class="form-control search-input">
		            <button type="submit" class="btn search-btn" value="Search"><i class="fas fa-search"></i></button>
		        </form>
            </div>
		    <nav id="docs-nav" class="docs-nav navbar">
			    <ul class="section-items list-unstyled nav flex-column pb-3">
				    <li class="nav-item section-title"><a class="nav-link scrollto active" href="#section-1"><span class="theme-icon-holder mr-2"><i class="fas fa-info"></i></span>Info</a></li>
				    <li class="nav-item"><a class="nav-link scrollto" href="#item-1-1">About us</a></li>
				    <li class="nav-item"><a class="nav-link scrollto" href="#item-1-2">Funding</a></li>
					<li class="nav-item"><a class="nav-link scrollto" href="#item-1-3">Development team</a></li>

				    <!-- <li class="nav-item"><a class="nav-link scrollto" href="#item-1-4">Portal overview</a></li> -->
					<li class="nav-item"><a class="nav-link scrollto" href="#item-1-5">Architecture</a></li>

				    <!-- <li class="nav-item section-title mt-3"><a class="nav-link scrollto" href="#section-2"><span class="theme-icon-holder mr-2"><i class="fas fa-folder-open"></i></span>Creating a study</a></li> -->
					<!-- <li class="nav-item"><a class="nav-link scrollto" href="#item-2-1">Uploading cfMeDIP-seq data</a></li> -->
				    <!-- <li class="nav-item"><a class="nav-link scrollto" href="#item-2-2">Uploading bin filters</a></li> -->
					<!-- <li class="nav-item"><a class="nav-link scrollto" href="#item-2-3">Uploading geneset data</a></li> -->
				    <!-- <li class="nav-item section-title mt-3"><a class="nav-link scrollto" href="#section-3"><span class="theme-icon-holder mr-2"><i class="fas fa-paper-plane"></i></span>Creating a run</a></li>
				    <li class="nav-item"><a class="nav-link scrollto" href="#item-3-1">Configuring parameters</a></li>
				    <li class="nav-item"><a class="nav-link scrollto" href="#item-3-2">Run status</a></li> -->
				    <!-- <li class="nav-item section-title mt-3"><a class="nav-link scrollto" href="#section-4"><span class="theme-icon-holder mr-2"><i class="fas fa-spinner"></i></span>Explore</a></li> -->
				    <!-- <li class="nav-item"><a class="nav-link scrollto" href="#item-4-1">Tables</a></li> -->
				    <!-- <li class="nav-item"><a class="nav-link scrollto" href="#item-4-2">Heatmaps</a></li> -->
				    <!-- <li class="nav-item section-title mt-3"><a class="nav-link scrollto" href="#section-5"><span class="theme-icon-holder mr-2"><i class="fas fa-arrow-down"></i></span>Export</a></li> -->
					<!-- <li class="nav-item"><a class="nav-link scrollto" href="#item-5-1">Full datasets</a></li> -->
					<!-- <li class="nav-item"><a class="nav-link scrollto" href="#item-5-2">Bin regions</a></li> -->
				    <!-- <li class="nav-item section-title mt-3"><a class="nav-link scrollto" href="#section-6"><span class="theme-icon-holder mr-2"><i class="fas fa-user-plus"></i></span>Sharing a project</a></li> -->
				    <!-- <li class="nav-item section-title mt-3"><a class="nav-link scrollto" href="#section-7"><span class="theme-icon-holder mr-2"><i class="fas fa-trash"></i></span>Deleting projects and runs</a></li> -->
					<li class="nav-item section-title mt-3"><a class="nav-link scrollto" href="#section-8"><span class="theme-icon-holder mr-2"><i class="fas fa-key"></i></span>Privacy and Security Policy</a></li>
					<!-- <li class="nav-item"><a class="nav-link scrollto" href="#item-8-1">Publicly uploaded information</a></li> -->
					<!-- <li class="nav-item"><a class="nav-link scrollto" href="#item-8-2">Security</a></li> -->
					<li class="nav-item"><a class="nav-link scrollto" href="#item-8-3">Personal/Protected Health Information (PHI)</a></li>
					<!-- <li class="nav-item"><a class="nav-link scrollto" href="#item-8-4">Private and Public projects</a></li> -->
					<li class="nav-item section-title mt-3"><a class="nav-link scrollto" href="#section-9"><span class="theme-icon-holder mr-2"><i class="fas fa-book-open"></i></span>Resources</a></li>
				    <!-- <li class="nav-item section-title mt-3"><a class="nav-link scrollto" href="#section-10"><span class="theme-icon-holder mr-2"><i class="fas fa-lightbulb"></i></span>FAQs</a></li> -->
			    </ul>

		    </nav><!--//docs-nav-->
	    </div><!--//docs-sidebar-->
	    <div class="docs-content">
		    <div class="container">
				<article class="docs-article" id="section-1">
				    <header class="docs-header">
						<!-- <h1 class="docs-heading">CReSCENT: <br/>CanceR Single Cell ExpressioN Toolkit</h1> -->
						<h1 class="docs-heading">								
							<span class="callout-icon-holder mr-1">
							<i class="fas fa-info-circle"></i>
						</span><!--//icon-holder-->
						Info 
						<span class="docs-time">Last updated: 2022-11-30 18:11 EST</span></h1>
					</header>
					<section class="docs-section" id='item-1-1'>
						<!-- <p><b>PLBR</b>: CanceR Single Cell ExpressioN Toolkit (<a href="https://crescent.cloud/" target="_blank" rel="noopener noreferrer">https://crescent.cloud</a>), is an <b>intuitive</b> and <b>scalable web portal</b> incorporating a containerized pipeline execution engine for <b>standardized analysis</b> of <b>single-cell RNA sequencing</b> (scRNA-seq) data. While scRNA-seq data for cancer specimens is easily generated, subsequent analysis requires high-performance computing infrastructure and user expertise to tailor interpretation for cancer. CReSCENT is populated with multiple <b>public datasets</b> and <b>preconfigured pipelines</b> that are accessible to computational biology non-experts, and user-editable to allow for optimization, comparison, and re-analysis on the fly.</p>
						<p>CReSCENT is supported and funded by Genome Canada, Medicine by Design, Princess Margaret Cancer Centre, the University Health Network, and the Hospital for Sick Children.</p>
					 -->

					<h2 class="section-heading">About us</h2>
					<div class="callout-block callout-block-warning">
						<div class="content">
							<h4 class="callout-title">
								<span class="callout-icon-holder mr-1">
									<i class="fas fa-exclamation-circle"></i>
								</span><!--//icon-holder-->
								VPN Protected!
							</h4>
							<p>Database portal (<a href="https://app.oncographer.ca/" target="_blank" rel="noopener noreferrer">app.oncographer.ca</a>) is currently only accessible using UHN VPN and with verified access to data.</p>
						</div><!--//content-->
					</div><!--//callout-block-->

					<p><b>OncoGrapher</b> (<a href="https://oncographer.ca/" target="_blank" rel="noopener noreferrer">oncographer.ca</a>) is a <b>database</b> and <b>scalable web portal</b> incorporating a graph-based architecture for standardized clinical abstraction using the MOHCCN standard.</p>
					<!-- <p><b>mCODER2</b>, a graph-based clinical abstraction tool (<a href="https://mcoder2.ca/" target="_blank" rel="noopener noreferrer">mcoder2.ca</a>), is a <b>database</b> and <b>scalable web portal</b> incorporating a graph-based architecture for <b>exploration</b> of <b>liquid biopsy data</b>. Liquid biopsy (the analysis of cell-free DNA to identify tumour signatures) represents a major advancement in cancer diagnostics and early detection. However, there is currently no centralized repository for the large amount of genetic and epigenetic data generated by these cell-free DNA screens. Thus, this valuable data is being siloed into small, disconnected research spaces, which puts significant drag on the pace of liquid biopsy development.</p> 
					 <p>To improve the accessibility of liquid biopsy data, and in particular fragmentomic profiling data, we present <b>PLBR</b> (<a href="https://mcoder2.ca/" target="_blank" rel="noopener noreferrer">mcoder2.ca</a>). PLBR will act as a centralized repository for fragmentomic data, thereby de-compartmentalizing the currently disjointed landscape of cfDNA screening data and allowing researchers to quickly and easily share this valuable data with their colleagues.</p>
							 
					 <p>Key to the detection and analysis of ctDNA have been development of genome-wide genetic and epigenetic signatures associated with specific cancer phenotypes. To enrich for, and further develop, such signatures in cell-free DNA multiple unbiased approaches have been developed to generate comprehensive DNA maps of fragmentation and copy number variation indicative of cancer-of-origin (shallow / low-pass whole genome sequencing), methylation indicative of gene silencing (e.g. cfMeDIP-seq, cell-free 5-methylcytosine DNA immunoprecipitation and high-throughput sequencing), hydroxymethylation indicative of gene activation (5hmC-Seal for cell-free 5-hydroxymethylcytosine), and mutational sampling indicative of tumour burden (deep >30X plasma WGS and gene panels).</p>
					 <p>Initially focused on basic and translational research for early detection and monitoring of disease, we will implement necessary data governance, authentication, and authorization controls to encourage uptake and contributions by clinical laboratories, as well as programs that may repurpose data beyond cancer with appropriate consent. </p>
					 <p>By centralizing and improving access to cfDNA screening data, the Phenomic Liquid Biopsy Resource represents a powerful synchronizing force between data generators and bioinformaticians. PLBR will facilitate the strong community of researchers at Princess Margaret in their efforts to gain new insights into cancer genomics via cfDNA analysis. In the short-term, this will enable numerous new collaborative research projects using the once-inaccessible wealth of fragmentomic data. In the long-term, the PLBR will result in improved detection of early-stage cancers as it facilitates further research into liquid biopsy technologies.</p> 
					  -->
					<!-- <div class="row">
							<div class="column">
								<a href="assets/images/logo.png" data-title="mCODER2" data-toggle="lightbox"><img class="figure-img img-fluid" src="assets/images/logo.png" alt="" style="width: 400px; height: 287px"></a>
							</div>
						</div> -->
					</section>
					<section class="docs-section" id="item-1-2">
						<h2 class="section-heading">Funding</h2>
						<p>We are looking for funding to support our development! Please contact us at <a href="mailto:mcoder@uhn.ca">mcoder@uhn.ca</a> to get in touch.</p>

						<!-- <p>PLBR is proudly supported and funded by the Canadian Cancer Society, the Computational Biology and Medicine Program, Princess Margaret Cancer Centre, and the University Health Network.</p> -->
						<!-- <div class="row">
							<div class="column">
								<a href="assets/images/ccs.png" data-title="Canadian Cancer Society" data-toggle="lightbox"><img class="figure-img img-fluid" src="assets/images/ccs.png" alt="" style="width: 500px"></a>

							</div>
							<div class="column">
								<a href="assets/images/cbmp.png" data-title="Computational Biology and Medicine Program" data-toggle="lightbox"><img class="figure-img img-fluid" src="assets/images/cbmp.png" alt="" style="width: 500px"></a>
							</div>
						</div> -->
					</section>
					<section class="docs-section" id="item-1-3">
						<h2 class="section-heading">Development team</h2>
						<!-- <img class="img-fluid" src="assets/images/intro_3_main_pages.jpg"> -->
						<!-- <figcaption class="figure-caption mt-3"><i class="fas fa-info-circle mr-2"></i>The three main pages within CReSCENT: 1) Projects/home, 2) Project runs, 3) Visualizations.</figcaption> -->
						<!-- <p><br/>The CReSCENT interface is represented by three main pages: Projects, Runs, and Visualizations. The Projects page (1) serves as the homepage, and here you can upload your own data or explore public cancer datasets curated by the CReSCENT team. Once a project is selected, the Runs page (2) shows all the runs/jobs that have been submitted. Run statuses are displayed by the run card colour (e.g. green for completed, yellow for in progress, etc.). The Runs page also houses functional buttons for sharing the current project with other CReSCENT users and uploading metadata to the project. Once a run is selected, the Visualizations page (3) displays the available visualizations, including quality control plots and explorable gene expression through t-SNE, UMAP, and violin plots. Actions within these three pages are detailed throughout this documentation.</p> -->
						<div class="row">
							<div class="column">
								<a href="assets/images/pugh.png" data-title="Pugh Lab" data-toggle="lightbox"><img class="figure-img img-fluid" src="assets/images/pugh.png" alt="" style="width: 425px"></a>
								<!-- <img src="assets/images/ccs.png" class="rounded float-left" alt="Snow" style="width:500px"> -->

							</div>
							<div class="column">
								<a href="assets/images/uhn.png" data-title="Princess Margaret Cancer Centre (UHN)" data-toggle="lightbox"><img class="figure-img img-fluid" src="assets/images/uhn.png" alt="" style="width: 575px"></a>
								<!-- <img src="assets/images/cbmp.png" class="rounded float-right" alt="Snow" style="width:500px"> -->
							</div>
						</div>
					</section><!--//section-->
				     <!-- <section class="docs-section" id="item-1-4"> -->
						<!-- <h2 class="section-heading">Portal overview</h2> -->
						<!-- <div class="callout-block callout-block-warning">
							<div class="content">
								<h4 class="callout-title">
									<span class="callout-icon-holder mr-1">
										<i class="fas fa-truck-loading"></i>
									</span>
									Under construction - coming soon!
								</h4>
							</div>
						</div> -->
						<!-- <img class="img-fluid" src="assets/images/intro_3_main_pages.jpg"> -->
						<!-- <figcaption class="figure-caption mt-3"><i class="fas fa-info-circle mr-2"></i>The three main pages within CReSCENT: 1) Projects/home, 2) Project runs, 3) Visualizations.</figcaption> -->
						<!-- <p><br/>The CReSCENT interface is represented by three main pages: Projects, Runs, and Visualizations. The Projects page (1) serves as the homepage, and here you can upload your own data or explore public cancer datasets curated by the CReSCENT team. Once a project is selected, the Runs page (2) shows all the runs/jobs that have been submitted. Run statuses are displayed by the run card colour (e.g. green for completed, yellow for in progress, etc.). The Runs page also houses functional buttons for sharing the current project with other CReSCENT users and uploading metadata to the project. Once a run is selected, the Visualizations page (3) displays the available visualizations, including quality control plots and explorable gene expression through t-SNE, UMAP, and violin plots. Actions within these three pages are detailed throughout this documentation.</p> -->
					<!-- </section> -->
					<section class="docs-section" id="item-1-5">
						<h2 class="section-heading">Architecture</h2>
						<a href="assets/images/mCODER2_Architecture.png" data-title="OncoGrapher Architecture" data-toggle="lightbox"><img class="figure-img img-fluid shadow rounded" src="assets/images/mCODER2_Architecture.png" alt="" style="width: 100%"></a>

						<!-- <img class="img-fluid" src="assets/images/plbr_architecture.png"> -->
						<!-- <figcaption class="figure-caption mt-3"><i class="fas fa-info-circle mr-2"></i>The three main pages within CReSCENT: 1) Projects/home, 2) Project runs, 3) Visualizations.</figcaption> -->
						<!-- <p><br/>The CReSCENT interface is represented by three main pages: Projects, Runs, and Visualizations. The Projects page (1) serves as the homepage, and here you can upload your own data or explore public cancer datasets curated by the CReSCENT team. Once a project is selected, the Runs page (2) shows all the runs/jobs that have been submitted. Run statuses are displayed by the run card colour (e.g. green for completed, yellow for in progress, etc.). The Runs page also houses functional buttons for sharing the current project with other CReSCENT users and uploading metadata to the project. Once a run is selected, the Visualizations page (3) displays the available visualizations, including quality control plots and explorable gene expression through t-SNE, UMAP, and violin plots. Actions within these three pages are detailed throughout this documentation.</p> -->
					</section><!--//section-->
			    </article><!--//docs-article-->
				

			    <!-- <article class="docs-article" id="section-2">
				    <header class="docs-header">
						
					    <h1 class="docs-heading">							
							<span class="callout-icon-holder mr-1">
							<i class="fas fa-folder-open"></i>
						</span>
						Creating a study</h1>
					    <section class="docs-intro"> -->
							<!-- <p>CReSCENT is organized into projects, each of which features a single scRNA-seq dataset. Projects exist within the portal as public or uploaded (private). Public projects are curated by the CReSCENT team, and results within these projects can be explored by all users. In addition to exploring these public projects, your own scRNA-seq data can be uploaded to be processed. To create a new project, click on the 'Uploaded Data' tab in the Projects page, followed by the '+' icon. A 'New Uploaded Project' window will appear where you can enter a name and description for the project. Associated scRNA-seq data is uploaded here.</p> -->
							<!-- <figure class="figure docs-figure py-3"> -->
								<!-- <a href="https://youtu.be/Ny-xUxQq7Z8" data-title="Uploading scRNA-seq data" data-toggle="lightbox"><img class="figure-img img-fluid shadow rounded" src="https://youtu.be/Ny-xUxQq7Z8" alt="" style="width: 800px;"></a> -->
								<!-- <iframe width="800" height="555" src="https://www.youtube.com/embed/Ny-xUxQq7Z8" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
								<figcaption class="figure-caption mt-3"><i class="fas fa-info-circle mr-2"></i>Quick walkthrough of how to upload your own scRNA-seq data and create a project in CReSCENT.</figcaption>
							</figure>	 -->

						<!-- </section>
				    </header>
				     <section class="docs-section" id="item-2-1">
						<h2 class="section-heading">Uploading cfMeDIP-seq data</h2>
						<div class="callout-block callout-block-warning">
							<div class="content">
								<h4 class="callout-title">
									<span class="callout-icon-holder mr-1">
										<i class="fas fa-truck-loading"></i>
									</span>
									Under construction - coming soon!
								</h4>
							</div>
						</div> -->

						<!-- <p>Single cell RNA-seq data must be uploaded to create a new project. A raw or normalized count matrix can be easily uploaded to the portal via the New Uploaded Project window. Once the data is uploaded, you can complete the project creation process by clicking ‘Create new project’. Accepted file formats for scRNA-seq data are further described below.</p> -->
						<!-- <h5>Upload example</h5> -->
						<!-- <figure class="figure docs-figure py-3"> -->
							<!-- <a href="assets/images/git_docs/1-create-proj/upload-projects.gif" data-title="Uploading scRNA-seq data" data-toggle="lightbox"><img class="figure-img img-fluid shadow rounded" src="assets/images/git_docs/1-create-proj/upload-projects.gif" alt="" style="width: 800px;"></a> -->
							<!-- <figcaption class="figure-caption mt-3"><i class="fas fa-info-circle mr-2"></i>Quick walkthrough of how to upload your own scRNA-seq data and create a project in CReSCENT.</figcaption> -->
						<!-- </figure>		 -->
						<!-- <div class="alert alert-warning"> -->
							<!-- <div class="content"> -->
								<!-- <h4 class="callout-title"> -->
									<!-- <span class="callout-icon-holder mr-1"> -->
										<!-- <i class="fas fa-truck-loading"></i> -->
									<!-- </span> -->
									<!-- Coming soon! -->
								<!-- </h4> -->
							<!-- </div> -->
						<!-- </div> -->
						<!-- <h3>Counts file sizes</h3>
						<p>Currently, the maximum file size that CReSCENT accepts is 200MB, which roughly corresponds to 100,000 cells from the 10X Cell Ranger count pipeline in the <a href="https://math.nist.gov/MatrixMarket/formats.html" target="_blank" rel="noopener noreferrer">Matrix Market (MTX) format</a> (described below). If your data comes from a different platform or is pre-normalized and contains decimals, you can considerably reduce its size by rounding RNA measurement values to 3 or 4 decimals.</p>
						</br>
						<h3>Counts file formats</h3>
						<p>CReSCENT currently accepts scRNA-seq data in the <a href="https://math.nist.gov/MatrixMarket/formats.html" target="_blank" rel="noopener noreferrer">MTX format</a> as input. Converted from gene-by-barcode text files (table below), the MTX format requires less storage space for sparse matrices where many elements are zeros. The three files used as input are the MTX (matrix.mtx.gz) and its barcode and gene/feature indices (barcodes.tsv.gz and features.mtx.gz, respectively). These files are one of the outputs from a <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/output/matrices" target="_blank" rel="noopener noreferrer">10X Cell Ranger</a> count run.</p>
						<h5>Gene-by-Barcode Matrix:</h5>
						<div class="table-responsive my-4">
							<table class="table table-striped">
								<thead>
									<tr>
										<th scope="col"></th>
										<th scope="col">BARCODE1</th>
										<th scope="col">BARCODE2</th>
										<th scope="col">BARCODE3</th>
									</tr>
								</thead>
								<tbody>
									<tr>
										<th scope="row">GENE1</th>
										<td>0</td>
										<td>1</td>
										<td>0</td>
									</tr>
									<tr>
										<th scope="row">GENE2</th>
										<td>2</td>
										<td>1</td>
										<td>7</td>
									</tr>
									<tr>
										<th scope="row">GENE3</th>
										<td>1</td>
										<td>4</td>
										<td>0</td>
									</tr>
								</tbody>
							</table>
						</div> -->
						<!-- <div class="callout-block callout-block-info">
                            <div class="content">
                                <h4 class="callout-title">
	                                <span class="callout-icon-holder mr-1">
		                                <i class="fas fa-info-circle"></i>
		                            </span>
	                                Note
	                            </h4>
								<p>One-line-command R script to convert gene-by-barcode text files to MTX files can be found in our <a href="https://github.com/pughlab/crescent/blob/master/bin/in_use/obtains_mtx_files_from_gene_x_barcode_matrix.R" target="_blank" rel="noopener noreferrer"><i class="fab fa-github"></i> GitHub</a>.</p>
							</div>
						</div> -->
						<!-- </br> -->
						<!-- <h3>Counts (raw) vs. pre-normalized data</h3> -->
						<!-- <p>CReSCENT is currently optimized to process raw counts from the 10X Cell Ranger count software. However, pre-normalized counts from other methods can be processed as long as they are provided in log scale, no negative values are included, and the QC filters are pre-computed (i.e. the Quality Control 'Apply QC Filters' parameter, found in the 'Pipeline' tab of the Visualizations page, should be set to 'No'). See <a href="#item-3-1">Configuring parameters</a> for more details </p> -->
					    <!-- </br> -->
						<!-- <div class="callout-block callout-block-info">
                            <div class="content">
                                <h4 class="callout-title">
	                                <span class="callout-icon-holder mr-1">
		                                <i class="fas fa-info-circle"></i>
		                            </span>
	                                Note
	                            </h4>
								<p>Example data from 10X Genomics is available! The 10X scRNA-seq dataset contains PBMCs (approximately 1000 cells). It can be downloaded from our portal when creating a new project or by clicking <a href="https://crescent.cloud/express/download/5e5818142f7290004cab4868" target="_blank" rel="noopener noreferrer">here</a>!</p>
							</div>
						</div> -->
						<!-- <div class="callout-block callout-block-warning">
                            <div class="content">
                                <h4 class="callout-title">
	                                <span class="callout-icon-holder mr-1">
		                                <i class="fas fa-bullhorn"></i>
		                            </span>
	                                Warning
	                            </h4>
								<p>Personal/protected health information (PHI) is prohibited from upload. Do not include patient names, dates of birth, MRNs and other patient identifiers in your uploads. Examples of PHI can be found <a href="https://privacyruleandresearch.nih.gov/pr_08.asp" target="_blank" rel="noopener noreferrer">here</a>. Only de-identified data is accepted in PLBR! </p>
							</div>
                        </div> -->

					<!-- </section> -->
					
					<!-- <section class="docs-section" id="item-2-2"> -->
						<!-- <h2 class="section-heading">Uploading bin filters</h2>	 -->
						<!-- <p>Following run completion, cell metadata can be uploaded, if available, via the Visualizations page in the Annotations tab. Cell metadata will be used to visualize and colour cells, as well as provide further insight into gene expression exploration in the Visualizations page. To upload metadata, click the 'Upload Metadata' button in the Annotations tab and then drag and drop your file to upload.</p> -->
						<!-- <h5>Upload example</h5> -->
						<!-- <figure class="figure docs-figure py-3">
							<a href="assets/images/git_docs/13-metadata/metadata.gif" data-title="Uploading cell metadata" data-toggle="lightbox"><img class="figure-img img-fluid shadow rounded" src="assets/images/git_docs/13-metadata/metadata.gif" alt="" style="width: 800px;"></a>
							<figcaption class="figure-caption mt-3"><i class="fas fa-info-circle mr-2"></i>Quick walkthrough of how to upload cell metadata for a project.</figcaption>
						</figure> -->
				
						<!-- <h3>Metadata file format</h3>
						<p>CReSCENT currently accepts cell metadata in a tab-separated file (tsv). We have adopted the <a href="https://github.com/broadinstitute/single_cell_portal/wiki/Metadata-File" target="_blank" rel="noopener noreferrer">Broad standard</a> for cell metadata. This tsv has cell barcodes in the first column and customized metadata labels in the subsequent columns (see table below). These labels can be categorized as "group" for categorical variables, like cell types, and "numeric" for continuous values, like orthogonal assay measurements. The double header is necessary to display the proper visualization with "NAME" and "TYPE" being mandatory. </p>
						<h5>Cell Metadata:</h5>
						<div class="table-responsive my-4"> -->
							<!-- <table class="table table-striped">
								<thead>
									<tr>
										<th scope="col">NAME</th>
										<th scope="col">Cell_Type</th>
										<th scope="col">Cell_Phase</th>
										<th scope="col">Mitochondrial_Fraction</th>
									</tr>
									<tr>
										<th scope="col">TYPE</th>
										<th scope="col">group</th>
										<th scope="col">group</th>
										<th scope="col">numeric</th>
									</tr>
								</thead>
								<tbody>
									<tr>
										<th scope="row">BARCODE1</th>
										<td>Endothelial_cell</td>
										<td>G1</td>
										<td>0.174</td>
									</tr>
									<tr>
										<th scope="row">BARCODE2</th>
										<td>Fibroblast</td>
										<td>S</td>
										<td>0.054</td>
									</tr>
									<tr>
										<th scope="row">BARCODE3</th>
										<td>Cardiac_muscle_cell</td>
										<td>G2</td>
										<td>0.199</td>
									</tr>
								</tbody>
							</table> -->
						<!-- </div> -->
						<!-- <div class="callout-block callout-block-info">
                            <div class="content">
                                <h4 class="callout-title">
	                                <span class="callout-icon-holder mr-1">
		                                <i class="fas fa-info-circle"></i>
		                            </span>
	                                Note
	                            </h4>
								<p>Metadata is an optional upload! Some examples include orthogonal assays, sample annotations, cell annotations, etc.</p>
							</div>
						</div> -->
						<!-- <div class="callout-block callout-block-warning">
							<div class="content">
								<h4 class="callout-title">
									<span class="callout-icon-holder mr-1">
										<i class="fas fa-truck-loading"></i>
									</span>
									Under construction - coming soon!
								</h4>
							</div>
						</div>

					</section>
			    </article>		     -->
			    
			    
			    <!-- <article class="docs-article" id="section-4">
				    <header class="docs-header">
					    <h1 class="docs-heading">
							<span class="callout-icon-holder mr-1">
								<i class="fas fa-spinner"></i>
							</span>
							
							Explore


						</h1>
						<div class="callout-block callout-block-warning">
							<div class="content">
								<h4 class="callout-title">
									<span class="callout-icon-holder mr-1">
										<i class="fas fa-truck-loading"></i>
									</span>
									Under construction - coming soon!
								</h4>
							</div>
						</div> -->
					    <!-- <section class="docs-intro"> -->
							<!-- <p>Once your run is submitted and completed, results are available to visualize and explore. There are a variety of interactive visualizations within CReSCENT. Quality control plots include violin plots of QC metrics before and after filtering as well as UMAP plots visualizing the different QC metrics with a gradient. Gene expression can be explored through t-SNE, UMAP, and violin plots. Along with the clustering results from the pipeline, cell metadata previously uploaded is incorporated to colour cells. </p> -->
							<!-- <figure class="figure docs-figure py-3"> -->
								<!-- <a href="assets/images/git_docs/7-results/results.gif" data-title="Viewing a completed run" data-toggle="lightbox"><img class="figure-img img-fluid shadow rounded" src="assets/images/git_docs/7-results/results.gif" alt="" style="width: 800px;"></a> -->
								<!-- <iframe width="800" height="555" src="https://www.youtube.com/embed/hRoficVJw1o" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe> -->
								<!-- <figcaption class="figure-caption mt-3"><i class="fas fa-info-circle mr-2"></i>Viewing a completed run in CReSCENT.</figcaption> -->
							<!-- </figure>					 -->
						<!-- </section> -->
				    <!-- </header>
				     <section class="docs-section" id="item-4-1">
						<h2 class="section-heading">Tables</h2>
						<div class="callout-block callout-block-warning">
							<div class="content">
								<h4 class="callout-title">
									<span class="callout-icon-holder mr-1">
										<i class="fas fa-truck-loading"></i>
									</span>
									Under construction - coming soon!
								</h4>
							</div>
						</div> -->
						<!-- <p>Quality control in the visualizations page consists of violin plots and UMAPs.</p>
						<h3>Quality control violin plots</h3>
						<p>The QC violin plots are interactive and display a variety of metrics. The four QC metrics (number of genes, number of reads, mitochondrial genes fraction, and ribosomal protein genes fraction) are displayed with distributions before and after QC filtering in the pipeline. </p>
						<figure class="figure docs-figure py-3">
							<a href="assets/images/git_docs/12-viz/qc/qc-violins.gif" data-title="Before/after filtering violin plots" data-toggle="lightbox"><img class="figure-img img-fluid shadow rounded" src="assets/images/git_docs/12-viz/qc/qc-violins.gif" alt="" style="width: 800px;"></a>
							<figcaption class="figure-caption mt-3"><i class="fas fa-info-circle mr-2"></i>Before/after filtering violin plots and how to switch between viewing barcodes and distribution statistics (mean, median, quartiles, etc.).</figcaption>
						</figure>	
						<h3>Quality control UMAP plots</h3>
						<p>The QC UMAP plots show the intensity of the four QC metrics with a gradient. </p>
						<figure class="figure docs-figure py-3">
							<a href="assets/images/git_docs/12-viz/qc/qc-umap.gif" data-title="QC UMAP Plots" data-toggle="lightbox"><img class="figure-img img-fluid shadow rounded" src="assets/images/git_docs/12-viz/qc/qc-umap.gif" alt="" style="width: 800px;"></a>
							<figcaption class="figure-caption mt-3"><i class="fas fa-info-circle mr-2"></i>QC UMAP plots showing distribution of the QC metrics across the cells.</figcaption>
						</figure>	 -->
					<!-- </section> -->
					
					<!-- <section class="docs-section" id="item-4-2">
						<h2 class="section-heading">Heatmaps</h2>
						<div class="callout-block callout-block-warning">
							<div class="content">
								<h4 class="callout-title">
									<span class="callout-icon-holder mr-1">
										<i class="fas fa-truck-loading"></i>
									</span>
									Under construction - coming soon!
								</h4>
							</div>
						</div> -->
						<!-- <p>Gene expression can be explored in the Visualizations page through t-SNE, UMAP, and violin plots. Gene expression can be explored by a) searching a gene of interest via the ‘Search Genes’ function or b) selecting a top differentially expressed gene (DEG) determined by CReSCENT’s pipeline which detects DEGs for each cell cluster vs. the rest of cells in the dataset. DEGs are detected using Seurat’s function <code>FindAllMarkers()</code> with most parameters set to defaults. Only genes detected in a minimum of 25% of cells (parameter <code>min.pct=0.25</code>) enter the DEG detection to speed up the function and skip genes that are infrequently expressed. Another parameter that is adjusted is a pseudocount to add to averaged expression values when calculating logFC (Seurat default=1); instead, we use <code>1 / number of cells in the dataset</code>, as suggested by <a href="https://f1000research.com/articles/7-1522" target="_blank" rel="noopener noreferrer">Inness and Bader (F1000Res, 2019)</a>.</p>
						<h3>t-SNE and UMAP</h3>
						<p>Both dimension reduction plots initially show cell clusters determined by CReSCENT's scRNA-seq pipeline. Cells can also be coloured by the previously uploaded metadata. Selecting a top DEG or searching a gene of interest will provide an opacity to the cells. The higher the normalized expression per cell, the darker the opacity. This gene exploration will provide insight into the expression of genes driving a cluster as well as potentially provide enough context to label the cluster.</p>
						<figure class="figure docs-figure py-3">
							<a href="assets/images/git_docs/12-viz/gene-expression/umap.gif" data-title="Gene Expression UMAP Plots" data-toggle="lightbox"><img class="figure-img img-fluid shadow rounded" src="assets/images/git_docs/12-viz/gene-expression/umap.gif" alt="" style="width: 800px;"></a>
							<figcaption class="figure-caption mt-3"><i class="fas fa-info-circle mr-2"></i>Gene expression UMAP plots showing cell metadata, cluster groups, and expression of genes across clusters.</figcaption>
						</figure>	
						<h3>Violin plots</h3>
						<p>The same gene expression exploration (cell metadata colouring, selecting a top expressed gene, and searching a gene of interest) as above can be done through violin plots. </p>
						<figure class="figure docs-figure py-3">
							<a href="assets/images/git_docs/12-viz/gene-expression/violins.gif" data-title="Gene Expression Violin Plots" data-toggle="lightbox"><img class="figure-img img-fluid shadow rounded" src="assets/images/git_docs/12-viz/gene-expression/violins.gif" alt="" style="width: 800px;"></a>
							<figcaption class="figure-caption mt-3"><i class="fas fa-info-circle mr-2"></i>Gene expression violin plots showing distributions of the expression of one gene across the clusters.</figcaption>
						</figure>	
						<h3>Dot plots</h3>
						<p>Gene expression can also be visualized as dot plots. Multiple genes (up to 20) can be selected through the top DEGs menu or by searching genes of interest. The x-axis displays the selected genes and the y-axis displays the groups (cell clusters determined by the CReSCENT pipeline or uploaded cell metadata). The size of the dots refer to the percentage of cells within a cluster that express a specific gene (i.e., cells expressing counts of a gene greater than 1 are included in this calculation). The opacity of the dots refer to the average expression of a specific gene across all clusters (i.e., the opacity is scaled based on the highest expression from all clusters for that gene). Hovering over a dot provides the total number of cells per cluster, the percentage of cells within the cluster expressing a gene, and the average gene expression across clusters.</p>
						<figure class="figure docs-figure py-3">
							<a href="assets/images/dot_plot.png" data-title="Gene Expression Dot Plots" data-toggle="lightbox"><img class="figure-img img-fluid shadow rounded" src="assets/images/dot_plot.png" alt="" style="width: 800px;"></a>
							<figcaption class="figure-caption mt-3"><i class="fas fa-info-circle mr-2"></i>Dot plots showing percentage of cells expressed per cluster (size) and average expression across clusters (opacity).</figcaption>
						</figure> -->
					<!-- </section>
			    </article> -->
			    
			    <!-- <article class="docs-article" id="section-5">
				    <header class="docs-header">
					    <h1 class="docs-heading">
							<span class="callout-icon-holder mr-1">
								<i class="fas fa-arrow-down"></i>
							</span>
							Export
						</h1>
						<div class="callout-block callout-block-warning">
							<div class="content">
								<h4 class="callout-title">
									<span class="callout-icon-holder mr-1">
										<i class="fas fa-truck-loading"></i>
									</span>
									Under construction - coming soon!
								</h4>
							</div>
						</div> -->

					    <!-- <section class="docs-intro">
							<p>Downloading results in CReSCENT is easy! After selecting a completed run, click the 'Results' tab in the Visualizations page and click the 'DOWNLOAD RESULTS' button to get a zipped folder of your run's results. This includes folders with QC plots and tables, dimension reduction plots and tables, cell cluster identity tables, average gene expression tables, differential gene expression tables and log files.</p>
							<figure class="figure docs-figure py-3"> -->
								<!-- <a href="assets/images/git_docs/10-download/download.gif" data-title="Downloading results of a completed run" data-toggle="lightbox"><img class="figure-img img-fluid shadow rounded" src="assets/images/git_docs/10-download/download.gif" alt="" style="width: 800px;"></a> -->
								<!-- <iframe width="800" height="555" src="https://www.youtube.com/embed/MgXnQ0RMQHI" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe> -->
								<!-- <figcaption class="figure-caption mt-3"><i class="fas fa-info-circle mr-2"></i>Downloading results of a completed run in CReSCENT.</figcaption> -->
							<!-- </figure>	 -->
							<!-- <div class="callout-block callout-block-info">
								<div class="content">
									<h4 class="callout-title">
										<span class="callout-icon-holder mr-1">
											<i class="fas fa-info-circle"></i>
										</span>
										Note
									</h4>
									<p>Downloaded results are not interactive CReSCENT plots! These are static plots and tables generated from the pipeline.</p>
								</div>
							</div>
						</section> -->
					<!-- </header> -->
					<!-- <section class="docs-section" id="item-5-1">
						<h2 class="section-heading">Full datasets</h2>
						<div class="callout-block callout-block-warning">
							<div class="content">
								<h4 class="callout-title">
									<span class="callout-icon-holder mr-1">
										<i class="fas fa-truck-loading"></i>
									</span>
									Under construction - coming soon!
								</h4>
							</div>
						</div> -->
						<!-- <p>The following run log files are provided within the <code>SEURAT/LOG_FILES</code> folder found in the download from CReSCENT:</p>
						<ul>
						    <li>UsedOptions.txt with the commands used for the one-line-command application.</li>
						    <li>RSessionInfo.txt with the list of R and R library versions used in the run.</li>
							<li>CPUusage.txt with the computing time used for each step of the run and the total time, also indicates the number of CPU cores used.</li>
						</ul>
						<div class="callout-block callout-block-info">
							<div class="content">
								<h4 class="callout-title">
									<span class="callout-icon-holder mr-1">
										<i class="fas fa-info-circle"></i>
									</span>
									Note
								</h4>
								<p>A link to the current version of the Rscript can be found in the Visualizations page under the 'Pipeline' tab. You can also find a copy of the one-line-command Rscript used in your run within your downloaded zip in the main directory.</p>
							</div>
						</div> -->
					<!-- </section> -->
				     <!-- <section class="docs-section" id="item-5-2">
						<h2 class="section-heading">Bin regions</h2>
						<div class="callout-block callout-block-warning">
							<div class="content">
								<h4 class="callout-title">
									<span class="callout-icon-holder mr-1">
										<i class="fas fa-truck-loading"></i>
									</span>
									Under construction - coming soon!
								</h4>
							</div>
						</div>
						<figure class="figure docs-figure py-3">
							<a href="assets/images/git_docs/8-failed-run-logs/fail-run.gif" data-title="Downloading logs of a failed run" data-toggle="lightbox"><img class="figure-img img-fluid shadow rounded" src="assets/images/git_docs/8-failed-run-logs/fail-run.gif" alt="" style="width: 800px;"></a>
							<figcaption class="figure-caption mt-3"><i class="fas fa-info-circle mr-2"></i>Downloading logs of a failed run and reviewing submitted parameters in CReSCENT.</figcaption>
						</figure>
					</section>
			    </article> -->
			    
				<article class="docs-article" id="section-8">
				    <header class="docs-header">
						<!-- <h1 class="docs-heading">CReSCENT: <br/>CanceR Single Cell ExpressioN Toolkit</h1> -->
						<h1 class="docs-heading">
							<span class="callout-icon-holder mr-1">
								<i class="fas fa-key"></i>
							</span><!--//icon-holder-->
							
							Privacy and Security Policy 
							
							<span class="docs-time">Last updated: 2022-10-28 13:31 EST</span></h1>
					</header>
					<!-- <section class="docs-section" id='item-8-1'>
						<h2 class="section-heading">Publicly uploaded information</h2>
						<p>Some features of CReSCENT are public facing and allow you to upload information that you may choose to make publicly available (see below). If you choose to upload content that is public-facing, third parties may access and use it. We do not sell any information that you provide to CReSCENT. However, any information that you make publicly available may be accessed and used by third parties, such as research organizations or commercial third parties.</p>
					</section>
					<section class="docs-section" id="item-8-2">
						<h2 class="section-heading">Security</h2>
						<p>The CReSCENT portal has security measures in place to protect the loss, misuse and alteration of the information under our control. CReSCENT, UHN or SickKids, however, are not liable for the loss, misuse or alteration of information on this site by any third party.</p>
					</section> -->
				     <section class="docs-section" id="item-8-3">
						<h2 class="section-heading">Personal/Protected Health Information (PHI)</h2>
						<p>PHI is any information in a medical record that can be used to identify an individual, and that was created, used, or disclosed in the course of providing a health care service, such as a diagnosis or treatment. This includes, but is not limited, to sample or metadata that can reveal the identity of a patient or donor. <b>PHI is not allowed in OncoGrapher.</b> Authors are responsible to make sure that PHI is not contained in the files that they upload. More examples of PHI can be found <a href="https://privacyruleandresearch.nih.gov/pr_08.asp" target="_blank" rel="noopener noreferrer">here</a>.</p>
						<div class="callout-block callout-block-warning">
                            <div class="content">
                                <h4 class="callout-title">
	                                <span class="callout-icon-holder mr-1">
		                                <i class="fas fa-bullhorn"></i>
		                            </span>
	                                Warning
	                            </h4>
								<p>Personal/protected health information (PHI) is prohibited from upload. Do not include patient names, dates of birth, MRNs and other patient identifiers in your uploads. Examples of PHI can be found <a href="https://privacyruleandresearch.nih.gov/pr_08.asp" target="_blank" rel="noopener noreferrer">here</a>. Only de-identified data is accepted in OncoGrapher! </p>
							</div>
                        </div>
					</section>
					
					<!-- <section class="docs-section" id="item-8-4">
						<h2 class="section-heading">Private and Public projects</h2>
						<p>A private project (and associated data) cannot be seen by anyone who has not been given access to the project by the authors (see <a href="#section-6">Sharing a project</a> for more details). In this way, it is possible to share the project to targeted individuals while keeping the project private to others. In contrast, a public project (and associated data) can be seen by and downloaded by anyone visiting the site. All newly created projects are created as private in CReSCENT.</p>
						<p>Authors who want to make their project(s) public must: </p>
						<ol>
							<li>Write to <a href="mailto:crescent@uhnresearch.ca">crescent@uhnresearch.ca</a> indicating that they want to make their Project(s) public and the project name(s) as they appear in the author’s Projects page under the 'Uploaded Data' tab.</li>
							<li>The CReSCENT team will evaluate the request for quality purposes, such as technical soundness of the datasets and reply back to the authors.</li>
							<li>Authors must accept this <b>Privacy and Security Policy</b>.</li>
						</ol>
					</section> -->
				</article>
				
				<article class="docs-article" id="section-9">
				    <header class="docs-header">
					    <h1 class="docs-heading">
							<span class="callout-icon-holder mr-1">
								<i class="fas fa-book-open"></i>
							</span><!--//icon-holder-->
							Resources
						</h1>
					    <section class="docs-intro">
							<p>OncoGrapher resources will be updated here.</p>
							<div class="col-md-6 col-12">
                                <ul class="list list-unstyled pl-0">
                                    <li><a href="https://app.oncographer.ca" target="_blank" rel="noopener noreferrer" class="btn btn-primary"><i class="fas fa-play-circle mr-2"></i> Go to app.oncographer.ca!</a></li>
									<!-- <li><a href="https://github.com/pughlab/crescent-frontend" target="_blank" rel="noopener noreferrer" class="btn btn-info"><i class="fab fa-github fa-fw"></i> View Web App Code</a></li>
									<li><a href="https://github.com/pughlab/crescent" target="_blank" rel="noopener noreferrer" class="btn btn-info"><i class="fas fa-code"></i> View Pipeline Scripts</a></li>
                                    <li><a href="https://doi.org/10.1093/nar/gkaa437" target="_blank" rel="noopener noreferrer" class="btn btn-secondary"><i class="fas fa-book mr-2"></i> View Publication</a></li>
                                    <li><a href="https://youtube.com/playlist?list=PLdXeBakJd4aLGpi2OPWTNeet2jb0iXfPJ" target="_blank" rel="noopener noreferrer" class="btn btn-danger"><i class="fab fa-youtube"></i> Youtube Tutorials</a></li>
									<li><a href="https://github.com/pughlab/crescent-frontend/issues" target="_blank" rel="noopener noreferrer" class="btn btn-warning"><i class="fas fa-bug mr-2"></i> Report Bugs</a></li> -->

								</ul>
                            </div>

						</section><!--//docs-intro-->
					</article>
				
			    <!-- <article class="docs-article" id="section-10">
				    <header class="docs-header">
					    <h1 class="docs-heading">FAQs</h1>
					    <section class="docs-intro">
							<h5 class="pt-3"><i class="fas fa-question-circle mr-1"></i>Does CReSCENT only support single dataset pipelines?</h5>
							<p>Currently, CReSCENT only has single dataset pipelines. We are currently working on an integrate datasets pipeline that will allow for multiple datasets and integration (i.e. clustering and differential expression) using the main datasets as anchors.</p>	
						</section>
				    </header>
			    </article> -->
				
			    <footer class="footer">
				    <div class="container text-center py-5">
								<!--/* This template is released under the Creative Commons Attribution 3.0 License. Please keep the attribution link below when using for your own project. Thank you for your support. :) If you'd like to use the template without the attribution, you can buy the commercial license via our website: themes.3rdwavemedia.com */-->
								<!-- <small class="copyright">Designed with <i class="fas fa-heart" style="color: #fb866a;"></i> by <a class="theme-link" href="http://themes.3rdwavemedia.com" target="_blank" rel="noopener noreferrer">Xiaoying Riley</a> for developers</small> -->
								<br>
								<!-- <small class="copyright">Template Copyright &copy; <a href="https://themes.3rdwavemedia.com/" target="_blank" rel="noopener noreferrer">3rd Wave Media</a></small> -->
				        <ul class="social-list list-unstyled pt-4 mb-0">
							<!-- <li class="list-inline-item"><a href="https://github.com/pughlab/crescent-frontend" target="_blank" rel="noopener noreferrer"><i class="fab fa-github fa-fw"></i></a></li> 
				            <li class="list-inline-item"><a href="https://doi.org/10.1093/nar/gkaa437" target="_blank" rel="noopener noreferrer"><i class="fas fa-book mr-2"></i></a></li>
				            <li class="list-inline-item"><a href="https://youtube.com/playlist?list=PLdXeBakJd4aLGpi2OPWTNeet2jb0iXfPJ" target="_blank" rel="noopener noreferrer"><i class="fab fa-youtube fa-fw"></i></a></li> -->
				        </ul><!--//social-list-->
				    </div>
			    </footer>
		    </div> 
	    </div>
    </div><!--//docs-wrapper-->
    
   
       
    <!-- Javascript -->          
    <script src="assets/plugins/jquery-3.4.1.min.js"></script>
    <script src="assets/plugins/popper.min.js"></script>
    <script src="assets/plugins/bootstrap/js/bootstrap.min.js"></script>  
    
    
    <!-- Page Specific JS -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.8/highlight.min.js"></script>
    <script src="assets/js/highlight-custom.js"></script> 
    <script src="assets/plugins/jquery.scrollTo.min.js"></script>
    <script src="assets/plugins/lightbox/dist/ekko-lightbox.min.js"></script> 
    <script src="assets/js/docs.js"></script> 

</body>
</html> 

